pkg/machine/wsl/declares.go

```diff
@@ -43,10 +42,16 @@ mkdir -p /home/[USER]/.config/systemd/[USER]/
 chown [USER]:[USER] /home/[USER]/.config
 `
 
+const configServicesPodman = `ln -fs /usr/lib/systemd/system/podman.socket /etc/systemd/system/sockets.target.wants/podman.socket` + configServices
+
 const sudoers = `%wheel        ALL=(ALL)       NOPASSWD: ALL
 `
 
 const bootstrap = `#!/bin/bash
+nohup bash -c 'while true; do sleep 60; done' >/dev/null 2>&1 &
```

Comment:
This is a side-effect of not being able to have the `user@1000` session start correctly... ?
```
systemctl status user@1000
```

This happened for me because I had some other distros running and this caused a conflict. Can you try to turn those off and then start again?